go.mod

```diff
@@ -44,6 +42,7 @@ require (
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/subosito/gotenv v1.2.0 // indirect
 	gopkg.in/ini.v1 v1.62.0 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
I haven't looked into what's bringing this in, hoping it's gogo/protobuf (since we're eliminating that in a separate PR)